docs/2.3StandardPayloadsLayout.md

```diff
@@ -1,17 +1,27 @@
+---
+description: Payloads may recursively contain other payloads, that enables the creation of higher-level protocols based on the same concepts of layers, as in traditional software and network architecture.
```

Comment:
```suggestion
description: This page describes the Standard Payloads Layout and different types of payloads that it includes.
```

Btw, I suggest my changes assuming that this is a description of what you find inside the page, not description of the page's subject.